lib/tasks/gds_to_cddo_tmp.rake

```diff
@@ -0,0 +1,19 @@
+desc "Migrate Service Standard Reports from GDS to CDDO "
+task gds_to_cddo_tmp: :environment do
+  content_ids = Republisher.content_id_and_locale_pairs_for_document_type("service_standard_report")
+
+  content_ids.each do |content_id, locale|
+    RepublishService.new.call(content_id, locale) do |payload|
+      gds_markdown = "From: | [Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service)"
```

Comment:
Is it OK for these to be `www.gov.uk` - how are they presented on the other environments?

Reply:
I think so yes, as it mirrors the current behaviour in the other test environments. In a perfect world these would be environment specific but I don't think that's possible to persist due to the data sync.